servicetalk-http-api/src/main/java/io/servicetalk/http/api/BaseHttpBuilder.java

```diff
@@ -73,14 +72,25 @@
 
     /**
      * Enables wire-logging for connections created by this builder.
-     * <p>
-     * All wire events will be logged at {@link Level#TRACE TRACE} level.
      *
      * @param loggerName The name of the logger to log wire events.
      * @return {@code this}.
      */
     public abstract BaseHttpBuilder<ResolvedAddress> enableWireLogging(String loggerName);
 
+    /**
+     * Enables wire-logging for connections created by this builder.
+     *
+     * @param loggerName The name of the logger to log wire events.
+     * @param logLevel The level to log at.
+     * @param logUserData {@code true} to include user data (e.g. data, headers, etc.). {@code false} to exclude this
+     * data.
```

Comment:
"to exclude this data" is a bit confusing because you do not know what to expect in this case. Consider clarifying: `{@code false} to exclude this data and log only network events`